src/wiz_l4.c

```diff
@@ -359,7 +359,7 @@ DEFINE_DO_FUN (do_oload) {
         obj_give_to_char (obj, ch);
     else
         obj_give_to_room (obj, ch->in_room);
-    send_to_char ("Ok.\n\r", ch);
+    printf_to_char(ch, "You created %p!\n\r", obj); 
```

Comment:
```
<1010hp 1000m 1000mv> load obj 3001
You created 0x7f3ccc236d70!
```
We can probably use an object name here instead ;)

Reply:
Resorted to using obj->short_desc

<20hp 100m 100mv> You created a bottle of beer!